exporters/otlp/otlplogs/otlplogshttp/client.go

```diff
@@ -306,8 +306,7 @@ func (d *httpClient) UploadLogs(ctx context.Context, protoLogs []*logspb.Resourc
 			}()
 		}
 
-		switch resp.StatusCode {
-		case http.StatusOK:
+		if resp.StatusCode/100 == 2 {
```

Comment:
I would suggest to align with upstream project and to avoid extra-calculation change it to:

```
switch sc := resp.StatusCode; {
  case sc >= 200 && sc <= 299:
```

ref: https://github.com/open-telemetry/opentelemetry-go/blob/main/exporters/otlp/otlpmetric/otlpmetrichttp/client.go#L156